### PR TITLE
removed '@' from the password validation schema

### PR DESCRIPTION
**Pull request by RASADSA**
fix for 
https://github.com/RasaHQ/rasa-x-helm/issues/221

### CODE REVIEW

**Comment by RASADSA** [comment]
@tczekajlo would merge it since this is a deployment timeout most likely imposed due rate limiting and this is one char :)

**Comment by tczekajlo** [comment]
Can you bump the chart version to `2.0.3` - https://github.com/RasaHQ/rasa-x-helm/blob/main/charts/rasa-x/Chart.yaml#L4 and add a changelog annotation? - https://github.com/RasaHQ/rasa-x-helm/blob/main/charts/rasa-x/Chart.yaml#L43-L45

🙏🏻

File: charts/rasa-x/Chart.yaml [hunk 1]
```diff
@@ -43,3 +43,5 @@ annotations:
   artifacthub.io/changes: |
     - kind: fixed
       description: Move the egress-from-rasa-x-to-https network policy out of if condition
+    - kind: fixed
+      description: added @ char to the not allowed chars for the rabbit password in helm chart values
```
Comment (tczekajlo) [inline charts/rasa-x/Chart.yaml:47]: No newline at end of file :)

File: charts/rasa-x/Chart.yaml [hunk 2]
```diff
@@ -43,3 +43,5 @@ annotations:
   artifacthub.io/changes: |
     - kind: fixed
       description: Move the egress-from-rasa-x-to-https network policy out of if condition
```
Comment (tczekajlo) [inline charts/rasa-x/Chart.yaml:45]:
```suggestion
```

**Review by tczekajlo — CHANGES_REQUESTED** [review]

**Review by tczekajlo — APPROVED** [review]
LGTM 🎖️

File: charts/rasa-x/Chart.yaml [hunk 3]
```diff
@@ -41,5 +42,7 @@ dependencies:
 annotations:
   # See: https://artifacthub.io/docs/topics/annotations/helm/#supported-annotations
   artifacthub.io/changes: |
+    - kind: fixed
+      description: added @ char to the not allowed chars for the rabbit password in helm chart values
     - kind: added
       description: Add an option to define hostNetwork and DNS policy for the rasa-x deployment
```
Comment (tczekajlo) [inline charts/rasa-x/Chart.yaml:48]:
```suggestion
```

File: charts/rasa-x/Chart.yaml [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ---
 apiVersion: v2
 
+
 version: "2.3.0"
```
Comment (tczekajlo) [inline charts/rasa-x/Chart.yaml:5]:
```suggestion
version: "2.3.1"
```

**Review by tczekajlo — CHANGES_REQUESTED** [review]

**Comment by RASADSA** [comment]
where is my mind... thx @tczekajlo 
image

**Review by tczekajlo — APPROVED** [review]

**Pull request merged**